### ISSUE TITLE
Réseaux

### ISSUE THREAD

**Issue by rcharb1** [pierre-H/optimiseur-coupe#61]
Je suis là

**Comment by pierre-H** [comment]
Génial !
Peux-tu faire un fork du projet ? Je n'y arrive pas

**Comment by rcharb1** [comment]
Je vais essayer tout de suite

**Comment by rcharb1** [comment]
Il est écrit "fork your own copy of Pierre-H/opti..." je ne crois pas qu'il s'agit de ça, n'est-ce pas ?

**Comment by pierre-H** [comment]
Normalement, il faut aller https://github.com/pierre-H/optimiseur-coupe et cliquer sur fork en haut à droite mais moi quand je clique je n'ai rien

**Comment by rcharb1** [comment]
oui, c'est ce que j'ai fait, et il me crée effectivement une version de mon projet : rcharb1/optimiseur etc

**Comment by pierre-H** [comment]
Quel est le lien exactement ?

**Comment by rcharb1** [comment]
https://github.com/rcharb1/optimiseur-coupe

**Comment by rcharb1** [comment]
mais je ne sais pas pourquoi il n'y a pas les issues...

**Comment by pierre-H** [comment]
https://programmers.stackexchange.com/questions/179468/forking-a-repo-on-github-but-allowing-new-issues-on-the-fork

**Comment by rcharb1** [comment]
il faut aussi que je l'actualise par rapport au dépôt origine car il est 97 commits en arrière !

**Comment by rcharb1** [comment]
pour les issues, c'est fait, merci pour le lien.

**Comment by pierre-H** [comment]
Il faut retirer xavier des collaborateurs

**Comment by rcharb1** [comment]
dans les settings aussi, je suppose, et ça peut être fait par moi, ou seulement le propriétairedu repo parent ?

**Comment by pierre-H** [comment]
oui

**Comment by rcharb1** [comment]
Ah, jusque là, il n'y avait que moi sur ce fork, maintenant, on est 2 et il n'y a donc pas besoin d'enlever qui que ce soit. Jepropose qu'on migre vers un issue du fork

**Comment by rcharb1** [comment]
tu as bien été ajouté au nouveau repo ?

**Comment by pierre-H** [comment]
Oui, mais il y a toujours Xavier, il recevra les mails...

**Comment by rcharb1** [comment]
reçu une notification ou non ?
